content/docs/command-reference/repro.md

```diff
@@ -97,6 +93,31 @@ up-to-date and only execute the final stage.
 
 ## Options
 
+- `targets` (optional argument) - one or more file or directory paths (to `.dvc`
+  or `dvc.yaml` files), or stage name(s) (`./dvc.yaml` by default). DVC will
+  reproduce them as detailed below.
+
+  - For **`dvc.yaml` files**, their [pipeline(s)](/doc/command-reference/dag)
+    are checked for changes, and reproduced as needed (explained in the command
+    [description](#description) above). E.g. `dvc repro pipes/linear/dvc.yaml`
+
+  - **Stage names** must be defined in `./dvc.yaml`. E.g.
+    `dvc repro train-vision`. Stages in other `dvc.yaml` files can be given
+    using by using a colon `:` following the path to that file. E.g.
+    `models/dvc.yaml:prepare`
+
+  - Files and directories tracked by **`.dvc` files** given as `targets` are
```

Comment:
let's skip this I think?

Reply:
Yes! I would like that too but are you sure? The help output (and Synopsis above) mention .dvc files so if we don't explain it here, it'd be up to the user to assume what will happen.

Reply:
Synopsis is a copy/paste from DVC which should probably change, so, let's not complicate. The point is not cover everything to the very small detail - the point is to focus on important stuff and convey it in the simplest possible format (e.g. people don't read long texts, unless they want to dive in)- thus short self-explainable examples are better

Reply:
Agree. Removed info about .dvc files from this ref.